src/AppRoot.tsx

```diff
@@ -1043,7 +1045,7 @@ class AppRoot extends React.Component<Props, State> {
                   updateVote: this.updateVote,
                   forceSaveVote: this.forceSaveVote,
                   userSettings,
-                  votes,
+                  votes: votes ?? {},
```

Comment:
Since `{}` is used three times, lets add some context by adding a variable to the top of the file and reference it three times:
```ts
const EmptyVotes = {}
```